lib/compiler/build.js

```diff
@@ -162,6 +166,23 @@ class Build extends CodeGenerator {
         this.emit(code);
         return node.uname;
     }
+    gen_native_proc_call(node) {
+        var code;
+        var proc = {
+            type: 'NativeModuleProc',
+            module: node.module_uname,
```

Comment:
I’d name this property `module_uname`. (The uname-less name confused me when reading `GraphCompiler` code before this part. All (or almost all) variables/properties which contain unames have `uname` somewhere in their names and I think it’s a good convention to prevent confusion with source-level names, which we also pass around all the time.)